<ui version="4.0" >
 <class>ClientMainClass</class>
 <widget class="QWidget" name="ClientMainClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>3</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Адрес сервера</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2" >
    <widget class="QLineEdit" name="leIPServer" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Локальный порт</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2" >
    <widget class="QLineEdit" name="leLocalPort" />
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Ник</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2" >
    <widget class="QLineEdit" name="leNik" />
   </item>
   <item row="3" column="0" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>75</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="2" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QPushButton" name="bEnter" >
       <property name="text" >
        <string>Войти</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bExit" >
       <property name="text" >
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="3" >
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>96</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
